bin/yaml-lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if ! command -v yamllint &> /dev/null
+then
+  pip install --user yamllint
+fi
+
+
+find ./addons -mtime -1 -name '*.yaml' -print0 | 
+	while IFS= read -r -d '' line; do
+		echo "$line"
+		# strip all the {{kurl }} directives out of the yaml before we lint it
+		sed -r 's/^\s*\{\{kurl[^\}]*\}\}\s*$//' < "$line" | yamllint - 
```

Comment:
IMO we shouldn't just remove kurl directives and instead replace them with "default" or similar

Reply:
another option would be to require that the yaml be valid even with kurl directives. for example using quotes for properties and comments for blocks.

```
# {{kurl- if and .IsJoin (not .IsUpgrade)}}
#   {{kurl- range .RemoteHosts}}
    - tcpConnect:
        collectorName: "weave {{kurl . }}"
        address: '{{kurl . }}:6783'
        timeout: 5s
#   {{kurl- end}}
# {{kurl- end}}
```

Reply:
exactly what I was thinking - and replacing `{{kurl . }}` with `default` would help ensure that